core/src/cloud/api.ts

```diff
@@ -591,6 +591,18 @@ export class CloudApi {
     })
   }
 
+  async put<T>(path: string, opts: ApiFetchOptions & { body?: any } = {}) {
+    const { body, headers, retry, retryDescription, maxRetries } = opts
+    return this.apiFetch<T>(path, {
+      method: "PUT",
+      body: body || {},
+      headers: headers || {},
+      retry: retry === true ? true : false, // defaults to false unless true is explicitly passed
```

Comment:
There must be a cleaner way to write this 🤔

How about
```suggestion
      retry: !!retry
```